### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by comfortablemilk4454** [Homebrew/discussions#4846] [Getting started]
### Output of `brew config`

```shell
idk what this means
```

### Output of `brew doctor`

```shell
idk what this means
```

### Description of issue

hi all im trying to install monkeydev and i dont know a thing about computers or coding and im just following the instructions listed in https://github.com/AloneMonkey/MonkeyDev/wiki/Installation and i get this error when i try to do the step that says "brew install ldid"~
<img width="896" alt="image" src="https://github.com/Homebrew/discussions/assets/43555540/f29a8a70-8368-42dd-a183-475bd6e9a9c0">
any help is much appreciated, im so lost and this is the only way i could find to talk to someone here, please explain what to do like i am 5 considering my lack of knowledge on this stuff! thank you so much and have a great day/night!

**Comment by p-linnane** [comment]
The instructions you are following are using some outdated functions that are no longer supported by brew. You will need to find another way to install sshpass.
